### PR TITLE
first draft of payment logic

### PR DESCRIPTION
**Pull request by set2k6**

### CODE REVIEW

File: bangazon.py [hunk 1]
```diff
@@ -102,7 +102,16 @@ def select_customer(self):
...
     self.page_clear()
-    print('option 3 - Create Payment Type')
+    print('Create your payment type.')
+    time.sleep(.5)
+    print('Enter your payment name.')
+    payment_name = input('Payment Name: ')
+    print('Enter your payment account number.')
+    payment_accountNum = input('Payment Account #: ')
+    new_payment = Payment(payment_name, payment_accountNum)
+    print('Payment type created.', new_payment.pay_uuid)
```
Comment (SimonHerrera) [inline bangazon.py]: delete this line

**Comment by ashwoodaa** [comment]
:+1:

File: bangazon.py [hunk 2]
```diff
@@ -102,7 +102,16 @@ def select_customer(self):
...
   def create_payment_type(self):
     self.page_clear()
-    print('option 3 - Create Payment Type')
+    print('Create your payment type.')
+    time.sleep(.5)
+    print('Enter your payment name.')
+    payment_name = input('Payment Name: ')
+    print('Enter your payment account number.')
+    payment_accountNum = input('Payment Account #: ')
+    new_payment = Payment(payment_name, payment_accountNum)
```
Comment (SimonHerrera) [inline bangazon.py:111]: self.all_customers[new_customer.cust_uuid] = new_customer

Comment (SimonHerrera) [inline bangazon.py:111]: below this line
self.serialize_data(self.all_customers, 'customers.p')
    time.sleep(1)

**Comment by keithbradley1** [comment]
nice code brah

**Pull request merged**